commands/faas.go

```diff
@@ -53,6 +53,7 @@ func Execute(customArgs []string) {
 	checkAndSetDefaultYaml()
 
 	faasCmd.SilenceUsage = true
+	faasCmd.SilenceErrors = true
 	faasCmd.SetArgs(customArgs[1:])
 	if err := faasCmd.Execute(); err != nil {
 		os.Exit(1)
```

Comment:
This is where the error message should be printed in the event of failure, can use `strings.Title()` to capitalise the error text (leave the error message without capitalisation).
```
if len(err.Error()) > 0 {
    fmt.Println(strings.Title(err.Error()))
}
```

Reply:
Wouldn't this capitalize the first letter in each word, including making `--username` into `--Username` and `-u` into `-U`?

Reply:
You're correct about this, my bad.